libs/shared/ui/src/lib/components/section/section.tsx

```diff
@@ -0,0 +1,42 @@
+import { type VariantProps, cva } from 'class-variance-authority'
+import { type ComponentPropsWithoutRef, type ElementRef, forwardRef, useContext } from 'react'
+import { twMerge } from 'tailwind-merge'
+import { LevelContext } from '../heading/level-context'
+
+const sectionVariants = cva(['flex', 'flex-col'], {
```

Comment:
I think it's not really necessary to have it, color and font size should be set on the children no? here it brings too much complexity

Reply:
https://github.com/Qovery/console/pull/855/commits/138fca8e3d98626ccb93f4d03efb8007c264e777